src/terraform/modules/policy-assignments/main.tf

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
`terraform fmt`

Reply:
Not sure the formatting changed but ran terraform fmt on file.